pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -560,7 +560,7 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 
 // updateDevicePath gets the node status to retrieve volume device path information.
 func (rc *reconciler) updateDevicePath(volumesNeedUpdate map[v1.UniqueVolumeName]*reconstructedVolume) {
-	node, fetchErr := rc.kubeClient.CoreV1().Nodes().Get(string(rc.nodeName), metav1.GetOptions{})
+	node, fetchErr := rc.kubeClients[0].CoreV1().Nodes().Get(string(rc.nodeName), metav1.GetOptions{})
```

Comment:
operations for node needs further work as it needs to use the rp client. this doesn't affect the current functionality and will be fixed in later PR.